### PR TITLE
support host parameter to datadog_event module

### PR DESCRIPTION
**Pull request by n0ts**
ISSUE TYPE

    datadog_event module does not correctly set host for event 

COMPONENT NAME

    monitoring/datadog_event.py

ANSIBLE VERSION

    latest

ref: https://github.com/ansible/ansible-modules-extras/pull/3125

### CODE REVIEW

File: lib/ansible/modules/monitoring/datadog_event.py [hunk 1]
```diff
@@ -152,8 +149,11 @@ def main():
 
 def _post_event(module):
     try:
+        if module.params['host'] is None:
+            module.params['host'] = platform.node().split('.')[0];
```
Comment (mattclay) [inline lib/ansible/modules/monitoring/datadog_event.py]: CI failure due to PEP 8 issue:

```
2017-02-01 07:09:22 ERROR: PEP 8: lib/ansible/modules/monitoring/datadog_event.py:153:66: E703 statement ends with a semicolon (current)
```

**Comment by mattclay** [comment]
CI failure due to missing `description` for the host option:

```
2017-02-02 00:27:28 ============================================================================
2017-02-02 00:27:28 lib/ansible/modules/monitoring/datadog_event.py
2017-02-02 00:27:28 ============================================================================
2017-02-02 00:27:28 ERROR: DOCUMENTATION.options.host.description: required key not provided @ data['options']['host']['description']. Got None
```

**Comment by n0ts** [comment]
shipit

**Comment by n0ts** [comment]
shipit

**Comment by n0ts** [comment]
shipt

**Comment by n0ts** [comment]
shipit

**Comment by n0ts** [comment]
shipit

**Comment by Akasurde** [comment]
shipit

**Review by Akasurde — APPROVED** [review]
LGTM

**Pull request merged**